apps/docs/content/components/forms/NumberField.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: NumberField
+description: A number field is a specialized number field that allows a user to enter a number.
+category: "forms"
+links:
+    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/inputs/src/NumberInputField.tsx
+---
+
+<Example src="inputs/docs/numberField/preview" isOpen />
+
+## Props
+
+<PropTable component="NumberField" />
+
+<FeatureFlag flag="next">
+    ## Guidelines
+
+    TODO: If we have some guidelines about this component&apos;s usage
+
+    ### Accessibility?
+
+    TODO: If we have some guidelines about this component and accessibility
+</FeatureFlag>
+
+## Anatomy
+
+<FeatureFlag flag="rc">
+### Concepts
+
+TODO: links to related concepts
+</FeatureFlag>
+
+### Composed Components
+
+A `NumberField` uses the following components.
+
+- [Label](./Label)
+- [Helper Message](./HelperMessage)
+- [Error Message](./ErrorMessage)
+
+## Examples
+
+### Disabled
+A number field in a disabled state shows that an input field exists, but is not available in that circumstance. This can be used to maintain layout continuity and communicate that a field may become available later.
+
+<Example src="inputs/docs/numberField/disabled" />
+
+### ReadOnly
+The `isReadOnly` prop makes the number field's text content immutable. Unlike `isDisabled`, the number field remains focusable and the contents can still be copied. See [the MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly) for more information.
```

Comment:
might not be right to say "text content" since only numbers are allowed. Can we say something else here?

Reply:
how about `makes the number field's content...`

Reply:
ya that should work.